packages/jsActions/nanoflow-actions-native/typings/mx.d.ts

```diff
@@ -11,5 +11,12 @@ declare namespace mx {
     }
     interface MxInterface {
         reload: () => void;
+        login2(
```

Comment:
Why is this needed? I don't see the original `login` here either 😕

Reply:
login exists in mendix-client typings.

Reply:
Yeah I don't know where that package comes from to be honest, but got it.

Reply:
It was published 5 years ago :D https://www.npmjs.com/package/mendix-client